admin/src/components/input/Coordinates.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div>
+    <b-form-group 
+      :label="$t('geo-coordinates.label')"
+      :invalid-feedback="$t('geo-coordinates.both-or-none')"
+      :state="isValid"
+      >
+      <b-form-group
+        :label="$t('latitude')"
+        label-for="home-community-latitude"
+        :description="$t('geo-coordinates.latitude.describe')"
+      >
+        <b-form-input
+          v-model="inputValue.coordinates[1]"
+          id="home-community-latitude"
+          type="text"
+          @input="splitCoordinates"
+        />
+      </b-form-group>
+      <b-form-group :label="$t('longitude')" label-for="home-community-longitude">
+        <b-form-input
+          v-model="inputValue.coordinates[0]"
+          id="home-community-longitude"
+          type="text"
+          @input="valueUpdated"
+        />
+      </b-form-group>
+    </b-form-group>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    value: Object,
+    default: () => ({
+      type: 'Point',
+      coordinates: [],
+    }),
+  },
+  data() {
+    return {
+      inputValue: this.value,
+    }
+  },
+  created() {
+    console.log("created value: %o, inputValue: %o", this.value, this.inputValue)
+  },
+  computed: {
+    isValid() {
+      return this.inputValue.coordinates.length === 0 || this.inputValue.coordinates.length === 2
+    }
+  },
+  methods: {
+    splitCoordinates(value) {
+      // format from google maps 'longitude, latitude'
+      const parts = value.split(',').map((part) => part.trim())
+
+      if (parts.length === 2) {
+        const [lat, lon] = parts
```

Comment:
Wie wird sichergestellt, dass array[0] = lat und array[1] = lon und nicht umgekehrt?

Reply:
Es ist das gebräuchliche Darstellungsformat in Deutsch und Englisch. [Quelle](https://de.wikipedia.org/wiki/Geographische_Koordinaten).
Das wird von google maps verwendet und sicherlich auch von anderen Karten-Apps.
Aber ich habe das jetzt nochmal anders gemacht, mit einem zusätzlichen Input-Feld. Siehe Screenshot.